media-tv/kodi/kodi-17.0_rc4.ebuild

```diff
@@ -56,7 +56,6 @@ COMMON_DEPEND="${PYTHON_DEPS}
 	>=dev-libs/yajl-2
 	dev-python/pillow[${PYTHON_USEDEP}]
 	dev-libs/libcdio
-	dvd? ( dev-libs/libcdio[-minimal] )
```

Comment:
The ebuild is renamed from rc3 to rc4. There is a version bump (not a rev bump).